lib/std/fmt.zig

```diff
@@ -1700,3 +1723,26 @@ test "positional with specifier" {
 test "positional/alignment/width/precision" {
     try testFmt("10.0", "{0d: >3.1}", .{@as(f64, 9.999)});
 }
+
+test "vector" {
+    if (builtin.os == .linux and builtin.arch == .arm and builtin.abi == .musleabihf) {
+        // TODO https://github.com/ziglang/zig/issues/3289
+        return error.SkipZigTest;
+    }
+
+    if (builtin.arch == .mipsel) {
+        return error.SkipZigTest;
```

Comment:
Can you link to the relevant issue here?

Reply:
Sadly not.
I noticed that the tests for mipsel did not pass in CI after initial commit of this PR.

Reply:
is it this one? https://github.com/ziglang/zig/issues/3317

Reply:
Let's see the CI results now :)